alvr/server_openvr/cpp/platform/linux/EncodePipelineNvEnc.h

```diff
@@ -30,5 +31,6 @@ class EncodePipelineNvEnc : public EncodePipeline {
     AVBufferRef* hw_ctx = nullptr;
```

Comment:
There's stuff that's created from the `vk_frame_ctx`, namely the `vk_frame`, meaning that `vk_frame_ctx` has to get destructed after `vk_frame. And since destruction happens bottom to top in declaration order it needs to look like this (maybe with ffmpeg specifically it also tolerates other orders, but I'm not taking chances):
```cpp
    Renderer* r = nullptr;
    std::unique_ptr<alvr::VkFrameCtx> vk_frame_ctx;
    AVBufferRef* hw_ctx = nullptr;
    std::unique_ptr<AVFrame, std::function<void(AVFrame*)>> vk_frame;
    AVFrame* hw_frame = nullptr;
```